README.md

```diff
@@ -1 +1,189 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Doctor Appoinment App <a name="about-project"></a>
+
+- Doctor Appointment App is a web-based application designed to streamline and simplify the process of scheduling and managing appointments with healthcare providers. This open-source project aims to provide a user-friendly interface for patients to book appointments, view their medical history, and receive reminders for upcoming visits. Additionally, it offers healthcare professionals a platform to manage their schedules and access patient information efficiently.
+
+- [Link to front end Repository](https://github.com/baqar-abbas/appointment-app-front-end)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add an Appoinment]**
+- **[Reserve an Appoinment]**
+- **[Delete an Appoinment]**
+- **[New user sign up functionality]**
+- **[New user login functionality]**
+- **[Admin user Add doctors]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo">https://doctor-app-u5x6.onrender.com/</a>
+
+- [Doctor Appoinment App]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get started with the app, you'll need to have Node.js installed on your system. Node.js is a JavaScript runtime that allows you to run JavaScript code outside of a web browser.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Ruby - The app is built on the Ruby programming language, so you'll need to have Ruby installed on your system. You can download Ruby from the official website.
+
+Ruby on Rails - Rails is a web application framework built on Ruby. You'll need to have Rails installed on your system to run the app. You can install Rails using the following command in your terminal:
+
+```sh
+  gem install rails
+```
+
+PostgreSQL - PostgreSQL is an open-source relational database management system used by the app to store and manage data. You'll need to have PostgreSQL installed on your system to run the app.
+
+### Setup
+
+First, clone the Github repository to your local machine. You can do this by running the following command in your terminal:
+
+```sh
+  git clone https://github.com/baqar-abbas/appointment-app-back-end
+```
+
+### Install
```

Comment:
- [ ] When I try to create the database according to instructions, this happens:

![image](https://github.com/baqar-abbas/appointment-app-back-end/assets/57421823/cce158ce-66a2-4cbe-8560-be17dc376335)

This is because the master.key is missing and your users will need to create it locally.

These are the steps:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:


![image](https://github.com/baqar-abbas/appointment-app-back-end/assets/57421823/66b1eace-db26-47ed-867b-23c1c77ba4d4)


![image](https://github.com/baqar-abbas/appointment-app-back-end/assets/57421823/50db45b1-ea8a-4cf5-9f7c-0a00c56ff640)


Kindly, include these instructions on your readme (or you can write your own), so your users are able to create their own `master.key` an be able to create the database in a correct manner. 👍

Reply:
Implemented suggested change screen shot attached below for your kind reference.

![image](https://github.com/baqar-abbas/appointment-app-back-end/assets/117091923/1a810f58-6d1d-4161-94fd-c6760c8aba76)
![image](https://github.com/baqar-abbas/appointment-app-back-end/assets/117091923/445e0adb-fa6a-4cc6-82d9-b2fd607ed7b8)